main/src/com/google/refine/LookupCacheManager.java

```diff
@@ -50,6 +49,8 @@
  */
 public class LookupCacheManager {
 
+    public static final String INDEX_COLUMN_NAME = "_OpenRefine_Index_Column_Name_";
```

Comment:
This seems unrelated. Is it intended to be included here? The change itself seems fine, but seems odd to have here.

Reply:
It looks like I should have explained the whole thing better, sorry. This change is intentional. Having this constant declared in `Cross` constitutes a dependency from `LookupCacheManager` (a core class) to `Cross` (a GREL function), which will prevent the separation of the GREL implementation as a separate Maven module which would depend on a core module.